Sources/Rownd/framework/Customizations.swift

```diff
@@ -32,12 +32,12 @@ open class RowndCustomizations: Encodable {
 
     open var sheetCornerBorderRadius: CGFloat = CGFloat(25.0)
 
-    open var loadingAnimation: Lottie.Animation? = nil
+    open var loadingAnimation: LottieAnimation? = nil
```

Comment:
Will this type work for cocoapods?

Reply:
hmm, good question. any reason why you think it might not?

Reply:
good catch. just tried to run a pod build and it failed for this reason

Reply:
Yeah I remember running into the same issue

Reply:
fixed by bumping the lottie version in podspec to 3.5.0